packages/jest-worker/src/createWorkerFarm.ts

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import {isAbsolute} from 'path';
+import WorkerFarm from './WorkerFarm';
+import type {WorkerFarmOptions, WorkerModule} from './types';
+
+export type JestWorkerFarm<T> = WorkerFarm & WorkerModule<T>;
+
+const reservedKeys = ['end', 'getStderr', 'getStdout', 'setup', 'teardown'];
+
+const isReserved = (methodName: string) => reservedKeys.includes(methodName);
+const isPrivate = (methodName: string) => methodName.startsWith('_');
```

Comment:
If a module is exporting a method, it is hardly private. Methods prefix with `_` are currently excluded, I just left this in place.